src/main/webapp/WEB-INF/index.html

```diff
@@ -68,6 +146,7 @@
 <script type="text/javascript" src="https://cdn.jsdelivr.net/momentjs/latest/moment.min.js"></script>
 <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/daterangepicker/daterangepicker.min.js"></script>
 <script src="/datepicker.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.7.6/handlebars.min.js"></script>
```

Comment:
nit: add `type="text/javascript"` for consistency. And also might be worth moving up one line just for nicer alignment.